compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/internal/CalendarModel.kt

```diff
@@ -298,22 +298,7 @@ internal data class DateInputFormat(val patternWithDelimiters: String, val delim
  * - dd.MM.yyyy
  * - MM/dd/yyyy
  */
-internal fun datePatternAsInputFormat(localeFormat: String): DateInputFormat {
```

Comment:
It's a common code that should be upsteamed. Please revert.

Reply:
Reverted. Will be upstreamed